Server/mods/deathmatch/logic/CGame.cpp

```diff
@@ -1410,6 +1410,16 @@ void CGame::QuitPlayer(CPlayer& Player, CClient::eQuitReasons Reason, bool bSayI
     // If he had joined
     if (Player.IsJoined())
     {
+        // Check if the player was inside a team and fire the event
+        if (Player.GetTeam())
```

Comment:
Use a variable to get and store the team just before your if statement instead of calling GetTeam twice.